make.ps1

```diff
@@ -38,7 +38,6 @@ function Clean-Command
 	{
 		$proc = Start-Process $msBuild $msBuildArguments -NoNewWindow -PassThru -Wait
 		rm *.dll
-		rm *.dll.config
```

Comment:
Stack overflow suggests something like this should work:
```powershell
Get-ChildItem *.dll.config -exclude OpenRA.Platforms.Default.dll.config | Remove-Item
```

Reply:
Looks like it does.